ngraph/frontend/onnx/frontend/src/op/matmul.hpp

```diff
@@ -13,14 +13,16 @@
 namespace ngraph {
 namespace onnx_import {
 namespace op {
+namespace detail {
+inline OutputVector matmul(const Output<ngraph::Node>& a, const Output<ngraph::Node>& b) {
```

Comment:
The `detail` namespace is usually put in cpp files to contain some helpers used in that file. You call this function from MatMulInteger implementation and this is where this helper should go. But in fact you can just create a MatMul op directly in there so IMO this helper is not needed at all.

Reply:
Removed redundant change, using `std::make_shared<default_opset::MatMul>()` directly now. Thank you!